specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/Patches.json

```diff
@@ -0,0 +1,542 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "description": "The filter to apply on the operation.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ARM operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/PatchCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patches_ListByBuilderResource_0": {
+            "$ref": "./examples/Patches_ListByBuilderResource.json"
+          }
+        },
+        "x-ms-odata": "#/definitions/PatchResource"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/builders/{builderName}/patches/{patchName}": {
+      "get": {
+        "tags": [
+          "Patches"
+        ],
+        "operationId": "Patches_Get",
+        "description": "Get details for specific patch by patch name.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./Builders.json#/parameters/BuilderNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PatchNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Patch information retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/PatchResource"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patches_Get_0": {
+            "$ref": "./examples/Patches_Get.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
>delete [](http://example.com/codeflow?start=7&length=6)

why the asymmetry - why do you support "delete" verb, but not "put"?

Reply:
For this one, because the resource is not created by user, and we still want to give the user the option to delete this resource, we support "delete" but not "put"

Reply:
How is the resource created then?